.gitignore

```diff
@@ -18,6 +18,7 @@ DartConfiguration.tcl
 .DS_Store
 *.egg-info
 *.egg
+env.yaml
```

Comment:
I'm curious why this was added.

Reply:
running the docker ci produces that file.  just getting vscode to inferior it